### ISSUE TITLE
Suggestion: Getting the element id

### ISSUE THREAD

**Issue by ejvindh** [sozi-projects/Sozi#660]
### Summary of your problem or feature request

Adding a feature to get the element id of a selected element

### Version of Sozi and other relevant software

I currently use Sozi22.09 on xubuntu. Inkscape 1.1

### Steps to reproduce the problem

Since the old Sozi13 doesn't work in Inkscape anymore (think it's because Inkscape doesn't accept Python2 anymore...), I've finally migrated fully to your rewritten standalone-tool. Yes, admittedly - it took me a while :-D

One of the things that made me hesitate for so long is that I liked very much the idea of Sozi focussing according to specific objects in the SVG-document. My presentation-SVG's are always filled with rectangles that determine the view throughout the presentation.

Now, after I was forced to migrate, I've found out that this feature is also available in the standalone-tool -- through the "Outline element Id". So now, I get the object-id from inkscape, enter it in the "Outline element Id", and click the wand. Et voila. 

This is really great, and it has helped me back on track with my use of your great tool.

My suggestion is: Would it somehow be possible within the standalone tool to select objects, and have their id's pasted into the "outline element id"-field? I'm uncertain whether it would be a huge job on a programming level, but it would ease the interface between Inkscape/Sozi a little.

I have my work-around, so don't put too much effort into it. But if it is an easy implementation, it would be helpful :-)

**Comment by aumouvantsillage** [comment]
> My suggestion is: Would it somehow be possible within the standalone tool to select objects, and have their id's pasted into the "outline element id"-field?

Yes. You can already do this by clicking on an element while pressing the Alt key.

**Comment by ejvindh** [comment]
That's great news!

It didn't work on my system, though (actually, I think I had already tried something like that). But I found out that this was due to the fact that on my Xubuntu "Alt-click" means "grab the window and move it" (also a problem if in Sozi's "Move"-mode, and I try to zoom).

But I never use that feature in XFCE, and after your post, I found out that it can be disabled:
https://askubuntu.com/a/270044

Now it's working! Thank you.
